client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -56,38 +56,43 @@ const HostingFeatures = () => {
 			? `/hosting-config/${ siteId }`
 			: `/overview/${ siteId }`
 	);
+	const hasRedirectQueryParam = searchParams.has( 'redirect_to' );
 	const hasEnTranslation = useHasEnTranslation();
 
-	const { data: siteTransferData, refetch: refetchSiteTransferData } = useSiteTransferStatusQuery(
-		siteId || undefined
-	);
+	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId || undefined, {
+		refetchInterval: 3000,
+		refetchIntervalInBackground: true,
+	} );
 	// `siteTransferData?.isTransferring` is not a fully reliable indicator by itself, which is why
 	// we also look at `siteTransferData.status`
 	const isTransferInProgress =
 		siteTransferData?.isTransferring || siteTransferData?.status === transferStates.COMPLETED;
```

Comment:
```suggestion
	const isTransferInProgress = siteTransferData?.isTransferring;
```
Coming back to this logic, it seems a bit odd. `isTransferring` is defined on the front-end, so it's easy to see when it changes:
https://github.com/Automattic/wp-calypso/blob/713982b209d2b83f701cc67e2ce65fa446b11678/client/landing/stepper/hooks/use-site-transfer/query.ts#L74

There's also the fact that `siteTransferData.status === transferStates.COMPLETED` clearly doesn't indicate that a transfer is **in progress**.

Should we change this to something like my suggestion?

Reply:
Actually it's my bad with naming. I have to rename it to `shouldRenderActivatingCopy`. Since there is the next case:
transfer is not in progress, it's already completed, but we haven't received info about `is_wpcom_atomic: true` and the button "Activate" would be rendered for a few seconds.
So, in general - we should not render  "Activate" button if the process of transferring is in progress or is *completed*.

Reply:
Definitely confusing, but I see what you're saying. It's unfortunate that `COMPLETED` doesn't actually mean "fully atomic and ready to go" in this instance. Renaming to `shouldRenderActivatingCopy` will help for sure, maybe we can also make the comment more explicit?

```suggestion
	// There can sometimes be a delay between `siteTransferData?.isTransferring` becoming
	// `false` and `isSiteWpcomAtomic` becoming `true`. This means we also need to hide
	// the activation button when the transfer  is `completed`, to cover the gap until
	// `isSiteWpcomAtomic` becomes true and hides the activation button on its own.
	const isTransferInProgress =
		siteTransferData?.isTransferring || siteTransferData?.status === transferStates.COMPLETED;
```

Reply:
Is the behavior to not show the activation button when a transfer is in either of these states something we can or want to add a test for? `TRANSFERRING` isn't something I think we're likely to introduce a regression around down the road, but the whole aspect of `COMPLETED` doesn't mean "completed" in this instance feels fragile if someone doesn't follow the logic (cognitive logic, not code logic 😉) down the road. Even with a note, better safe than sorry?

Reply:
> Since there is the next case: transfer is not in progress, it's already completed, but we haven't received info about is_wpcom_atomic: true and the button "Activate" would be rendered for a few seconds.

Gotcha. Let's rename the variable, then 👍

Also, we could maybe change the value to something like this:

```tsx
const shouldRenderActivatingCopy  = siteTransferData?.status && siteTransferData.status !== transferStates.NONE;
```

This component won't render at all once the Atomic transfer is complete, so what we really want to know here is whether an Atomic transfer was ever initiated.